README.md

```diff
@@ -1,222 +1,166 @@
 
-```bash
-npm run rename
-```
+    ```bash
+    bash bin/setup.sh
+    ```
 
-This will make changes to theme name, description, author, text domain, package, namespace, and constants (this is important when specifying environment variable).
+## Development Start
 
-## Development Pre Start
+1. Builds assets in watch/development mode using Webpack:
 
-After renaming your theme, run this to setup WordPress on the server.
-The script will install `npm` and `composer` dependencies and install the latest version of WordPress.
+    ```bash
+    npm start
+    ```
 
-```bash
-bash bin/setup.sh
-```
+2. Update your wp-config.php file with project specific configuration that you can tailor according to your project needs. Compare your `wp-config.php` file with this example and update accordingly:
 
-After running setup script, you'll need to create `wp-config.php`. You can do that manually, or use WP-CLI
+    ```php
+    // Must be set.
+    // Possible options are develop, staging and production.
+    define( 'INF_ENV', 'develop' );
 
-```bash
-wp config create --dbname={DBNAME} --dbuser={DBUSER} --dbpass={DBPASS}
-wp core install --url={dev.boilerplate.com} --title={THEMENAME} --admin_user={ADMINUSER} --admin_email={ADMINMAIL}
-wp theme activate {THEMENAME}
-```
+    /* That's all, stop editing! Happy blogging. */
 
-## Development Start
+    /** Absolute path to the WordPress directory. */
+    if ( !defined('ABSPATH') )
+      define('ABSPATH', dirname(__FILE__) . '/');
 
-Builds assets in watch mode using Webpack.
+    // Include wp config for your project.
+    require_once(ABSPATH . 'wp-config-project.php');
 
-```bash
-npm start
-```
+    /** Sets up WordPress vars and included files. */
+    require_once(ABSPATH . 'wp-settings.php');
+    ```
 
-## Browser sync
+    * Remove `define( 'WP_DEBUG', false );`. This is defined in the `wp-config-project.php` file.
 
-We are using BrowserSync to sync assets and enable easy cross-device testing.
-It is tested on MAMP and Vagrant (VVV).
+    * Set `INF_ENV` variable to correspond with your environment. That will add global setup for logging errors, disabling auto-update, some optimizations and it will change the color of admin so it is easier for you to know on what environment you are editing.
 
-## Linting Assets (JS,SASS)
 
-Lints JS and SASS using Webpack
+3. You have BrowserSync to sync assets and enable easy cross-device testing. Once BrowserSync is active you will get url in the terminal. Usualy it is `localhost:3000`.
 
-```bash
-npm run precommit
-```
+## Liniting Files
 
-## Linting PHP ##
+1. **Linting assets** - JS and SASS using Webpack:
 
-We are using [Infinum coding standards for WordPress](https://github.com/infinum/coding-standards-wp) to check php files.
+    ```bash
+    npm run precommit
```

Comment:
Can we setup `githooks` here instead of manually running scripts? [Githooks](https://githooks.com/)

Reply:
sure we can add that after we test it out. lets set this for version 3.1?